src/cocaine-app/sched/__init__.py

```diff
@@ -219,15 +219,15 @@ def cancel_crossing_jobs(self, job_type, sched_params, demand):
                     # These jobs could be cancelled, but there is a potential race between making decision
                     #  and actual job status. Since we are prohibited to cancel running jobs of not STOP_ALLOWED_TYPES
                     # skip cancellation for a while. That would be fixed as soon as job queries would be introduced
-                    logger.info('Job was not running {} of type {}'.format(job.id, job.type))
+                    logger.debug('Job was not running {} of type {}'.format(job.id, job.type))
```

Comment:
fixed in https://github.com/yandex/mastermind/pull/436/commits/cf88e8cbbb742dbc62fc23a517fd757c276c070d